### DISCUSSION TITLE
New release? caffeine had too many dependencies

### DISCUSSION THREAD

**Discussion by ralmeida-espatial** [nielsbasjes/yauaa#913] [Q&A] [answered]
commit [94fd08a](https://github.com/nielsbasjes/yauaa/commit/94fd08a41272d74cff46602c7bffa26f4e40a294) already updated caffeine version that was having too many dependencies (https://github.com/ben-manes/caffeine/discussions/867). Was just wondering if that would entail a 7.12.1? :) Was trying to use the latest yauaa but too many conflicts because of the issue with caffeine...

**Comment by nielsbasjes** [comment]
Sounds like a good reason for a new release.

**Reply by nielsbasjes** [reply]
Release is going slower than expected:

`Uploaded to sonatype: ... (5.2 MB at 23 kB/s)`

**Comment by nielsbasjes** [comment, marked as answer]
Released
https://repo1.maven.org/maven2/nl/basjes/parse/useragent/yauaa/7.12.1/

**Reply by nielsbasjes** [reply]
@ralmeida-espatial I would love to hear if this fixes the problems you mentioned.

**Reply by ralmeida-espatial** [reply]
yes, no more dependencies issues :) Thank you
